addons/sys_core/initSettings.sqf

```diff
@@ -106,6 +106,15 @@
     {[_this, true] call EFUNC(api,ignoreAntennaDirection)} // @todo remove second parameter in 2.7.0
 ] call CBA_Settings_fnc_init;
 
+// Antena direction
+[
+    QGVAR(automaticAntennaDirection),
+    "CHECKBOX",
+    localize LSTRING(autoAntennaDirection_displayName),
+    "ACRE2",
+    false
```

Comment:
Call the "set antenna dir" function when the setting changes? In cases where it should switch automatically, but doesn't because you just changed the setting?

Reply:
The variable `acre_sys_core_automaticAntennaDirection` is checked in `acre_sys_components_fnc_getAntennaDirMan` (which is called for every calculation run). So there is basically not function to set the antenna direction in automatic mode.